keywords/MobileRestClient.py

```diff
@@ -1761,6 +1761,34 @@ def add_design_doc(self, url, db, name, doc):
             resp_obj = resp.json()
             return resp_obj["id"]
 
+    def get_design_doc_rev(self, url, db, name):
+        """
+        Keyword that gets a Design Doc revision
+        """
+        resp = self._session.get("{}/{}/_design/{}".format(url, db, name))
+        log_r(resp)
+        resp.raise_for_status()
+
+        # Only return a response if adding to the listener
+        # Sync Gateway does not return a response
+        if self.get_server_type(url) == ServerType.listener:
```

Comment:
SG does not return a value:
» http put <SG_IP>:4985/sg_db1/_design/dd content_view='{"map":"function(doc, meta){if(doc.content){emit(doc._id,doc._rev);}}"}}}'
HTTP/1.1 201 Created
Content-Length: 0
Content-Type: text/plain; charset=utf-8
Date: Wed, 03 May 2017 22:17:06 GMT
Server: Couchbase Sync Gateway/1.4.1

» http get<SG_IP>:4985/sg_db1/_design/dd           
HTTP/1.1 200 OK
Content-Length: 2
Content-Type: application/json
Date: Wed, 03 May 2017 22:19:58 GMT
Server: Couchbase Sync Gateway/1.4.1

{}